cram_pr2/cram_pr2_projection/src/tf.lisp

```diff
@@ -86,24 +86,26 @@
              (urdf-link-names (remove-if (lambda (x) (member x bullet-links))
                                          (loop for name being the hash-keys in urdf-links
                                                collecting name))))
-        (let ((prev-link-count))
+        (let ((prev-link-count 0))
           ;; while there are still links left to be asserted in tf
           (loop while (> (length urdf-link-names) 0)
                 do ;; if no links were asserted last loop, we are stuck and have to return
-                   (if (eq prev-link-count (length urdf-link-names))
+                   (if (= prev-link-count (length urdf-link-names))
                        (progn
-                           (roslisp:ros-error (pr2-proj tf) "Somehow a virtual link with no parents in the bullet world was found in the urdf. This should never happen.")
+                           (roslisp:ros-error (pr2-proj tf) "Somehow a virtual link with no parents in ~
+						   the bullet world was found in the urdf. This should never happen.")
                            (return))
                        (setf prev-link-count (length urdf-link-names)))
-                   (let ((next-round (list)))
+                   (let (next-round)
                      ;; try to assert a transform from a parent in the bullet world to the virtual links
                      (loop for link-name in urdf-link-names
                            do (let* ((link-joint (cl-urdf:from-joint (gethash link-name urdf-links)))
                                      (parent-name (cl-urdf:name (cl-urdf:parent link-joint))))
                                 (when (and (not (eq (cl-urdf:joint-type link-joint) :FIXED))
                                            display-warnings)
                                   (roslisp:ros-warn (pr2-proj tf)
-                                                    "Joint of ~a is ~a. Only links on FIXED joints should be asserted as virtual links."
+                                                    "Joint of ~a is ~a. Only links on FIXED joints should ~
+													be asserted as virtual links."
```

Comment:
It might be just the GitHub visualization but the line is still way too long (> 80 characters). If you remove more whitespace before the second line it will get shorter.